File: kalkulaator/src/main/java/ee/ken/kalkulaator/Toiduaine.java
package ee.ken.kalkulaator;

public class Toiduaine {
    String nimi;
    int rasv;
    int sysivesik;
    int valk;

    public String getNimi() {
        return nimi;
    }

    public int getRasv() {
        return rasv;
    }

    public int getSysivesik() {
        return sysivesik;
    }

    public int getValk() {
        return valk;
    }

    public Toiduaine(String nimi, int rasv, int sysivesik, int valk) {
        this.nimi = nimi;
        this.rasv = rasv;
        this.sysivesik = sysivesik;
        this.valk = valk;
    }
}

File: auto/auto/src/pages/Toiduaine.js
import React, { useEffect, useState } from 'react'

function Toiduaine() {
    const[toiduaine, setToiduaine] = useState([])
    const[vastus, setVastus] = useState()

    useEffect(() => {
        fetch("http://localhost:8080/toiduained")
        .then(res => res.json())
        .then(json => setToiduaine(json))
      }, []);

  return (
    <div>
        {toiduaine.map(toiduaine =>
            <div>
                <div><b>{toiduaine.nimi}</b></div>
                <div>Valgu sisaldus: {toiduaine.valk}</div>
                <div>Süsivesikute sisaldus: {toiduaine.sysivesik}</div>
                <div>Rasva sisaldus: {toiduaine.rasv}</div><br/>
                </div>
        )}
    </div>
  )
}

export default Toiduaine